app/views/admin/event_previews/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<% provide(:custom_css, 'admin/event_previews/index') %>
+
+<section id="admin-preview-index" class="section-container">
+  <div class="row">
+    <div class="title">
+      Event Preview for NUANSA <%= @event.year %>
+    </div>
+    <div class="content">
+      <div class="col-lg-12">
+        <div class="row">
+          <div class="col-md-6">
+            <%= button_to("Back to event", admin_event_path(@event.year), method: "get", class: "btn btn-info") %>
+          </div>
+          <div class="col-md-6">
+            <%= button_to("Upload new image", new_admin_event_event_preview_path(@event.year), method: "get", class: "btn btn-info") %>
```

Comment:
It would be better to write this in two lines for better readability, because it doesn't look good on web like github if you write too long in one line.